app/uk/gov/hmrc/nisp/views/page_not_found_template.scala.html

```diff
@@ -0,0 +1,41 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.partials.FormPartialRetriever
+@import uk.gov.hmrc.nisp.views.viewParams.MainTemplateParams
+@import play.api.Application
+@import uk.gov.hmrc.renderer.TemplateRenderer
+
+
+@()(implicit lang: Lang, request: Request[_], messages: Messages, application: Application, formPartialRetriever: FormPartialRetriever, templateRenderer: TemplateRenderer)
+
+@anchor(text: String, url: String) = {<a href="@url" target="_blank">@text</a>}
+
+
+@unauthenticatedMain(MainTemplateParams(pageTitle = Some(Messages("global.page.not.found.error.title")))) {
+
+<p>@Messages("global.page.not.found.error.check.web.address.correct")</p>
```

Comment:
Swap all `@Messages` for `@messages` to use the implicit messages you have provided to the view.